app/views/matches/_edit_results.html.erb

```diff
@@ -22,12 +22,19 @@
               <%= builder.number_field :score_away, in: -2e9..2e9, step: 1, class: 'form-control' %>
             </div>
           </div>
+          <% unless builder.object.new_record? %>
+            <div class="form-group">
+              <% if can? :confirm, builder.object %>
+                <%= link_to t("view.match.confirm_game_result"), confirm_game_result_match_path(id: @match.id, result_id: builder.object.id), class: 'btn btn-default' %>
```

Comment:
I would add `col-xs-offset-1` so the layout looks better

Reply:
Cool :)